docker-compose-sunbird/imports/nginx/nginx.conf

```diff
@@ -1,5 +1,5 @@
 server {
-    listen 80;
+    listen 8000;
```

Comment:
lets keep 80 as its standard practice

Reply:
@vishwa-vyom had asked this to be updated to a non-privileged port earlier in an internal discussion, should I revert this change?

Reply:
this will be used to route internally so let this be 80